<launch>
<node pkg="rosserial_python" type="serial_node.py" name="arduino0" output="screen">
<param name="port" value="/dev/ttyS0"/>
<param name="baud" value="115200"/>
</node>
<include file="$(find mavros)/launch/px4.launch">
<arg name="fcu_url" value="/dev/ttyUSB0:921600"/>
</include>
<node pkg="trims" type="checker" name="checker" output="screen"/>
<node pkg="trims" type="engUnits" name="engUnits" output="screen"/>
<node pkg="rosbag" type="record" name="record" output="screen" args="-ao /home/quad/bag_files/auto_bags/QRex"/>
</launch>
